checkpoint-single-nic/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
```

Comment:
This should be "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",